src/components/common/modal/modals.tsx

```diff
@@ -17,10 +16,7 @@ export const Modals = () => {
     const renderModalContent = (modal: TModalProps, index: number) => {
         switch (modal?.type) {
```

Comment:
add a check to prevent this function from running if the type is undefined or null, otherwise it will print console.error()

Reply:
updated in the latest changes